01-understand-network/Starter/app/src/main/java/com/yourcompany/android/moviediary/ui/home/HomeScreen.kt

```diff
@@ -28,7 +28,7 @@
  * THE SOFTWARE.
  */
 
-package com.yourcompany.android.moviediary.ui.movies
```

Comment:
⬆️

Reply:
This is from an old commit, you should see the final changes here: https://github.com/kodecocodes/m3-nra-materials/pull/2/files